frontend/src/features/properties/map/acquisition/add/AcquisitionProperties.tsx

```diff
@@ -1,11 +1,17 @@
 import MapSelectorContainer from 'features/properties/selector/MapSelectorContainer';
 import { IMapProperty } from 'features/properties/selector/models';
-import { FieldArray, useFormikContext } from 'formik';
+import { FieldArray, FormikProps, useFormikContext } from 'formik';
 import { Col, Row } from 'react-bootstrap';
 
 import { AcquisitionForm, AcquisitionPropertyForm } from './models';
 
-export const AcquisitionProperties: React.FunctionComponent = () => {
+interface AcquisitionPropertiesProp {
+  formikProps: FormikProps<AcquisitionForm>;
+}
+
+export const AcquisitionProperties: React.FunctionComponent<AcquisitionPropertiesProp> = ({
+  formikProps,
+}) => {
   const { values } = useFormikContext<AcquisitionForm>();
```

Comment:
@Maheshbabu28 please either pass the formik props or use useFormikContext, not both.

Reply:
done